slash-commands/moderation/ban.js

```diff
@@ -0,0 +1,111 @@
+const Discord = require('discord.js');
+const {SlashCommandBuilder} = require('@discordjs/builders');
+const {promisePool} = require('../../config/db');
+const {sendToAuditLogsChannel} = require('../../helpers/sendToAuditLogs');
+const {
+  isServerStaff,
+  sendNoTargetStaffReply,
+} = require('../../helpers/validation');
+const {verifyReasonLength} = require('../../helpers/stringHelpers');
+
+const banIntro = "You've been banned for the following reason: ```";
+const unbanRequest =
+  ' ``` If you wish to challenge this ban, please submit a response in this Google Form: https://forms.gle/KxTMhPbi866r2FEz5';
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('ban')
+    .setDefaultPermission(false)
+    .setDescription('Bans a user')
+    .addUserOption((option) =>
+      option.setName('target').setDescription('The user').setRequired(true)
+    )
+    .addStringOption((option) =>
+      option.setName('reason').setDescription('The reason').setRequired(true)
+    )
+    .addIntegerOption((option) =>
+      option
+        .setName('days')
+        .setDescription(
+          'Number of days of messages to delete (between 0 and 7)'
+        )
+        .setRequired(true)
+    ),
+  async execute(interaction) {
+    const targetUser = await interaction.options.getUser('target');
+
+    if (await isServerStaff(interaction, targetUser)) {
+      return await sendNoTargetStaffReply(interaction);
+    }
+
+    const guildBans = await interaction.guild.bans.fetch();
+    if (guildBans.has(targetUser.id)) {
+      return await interaction.reply('The user is already banned');
+    }
+
+    const reason = interaction.options.getString('reason');
+    if (!verifyReasonLength(reason, interaction)) return;
+
+    const days = interaction.options.getInteger('days');
+    if (days < 0 || days > 7) {
+      return await interaction.reply(
+        'The number of days are not between 0 and 7'
```

Comment:
Should specify that it's between 0 and 7, inclusive.

Reply:
I made this change